PRS12001-13000/PRS12002Gabrami.xml

```diff
@@ -40,17 +40,43 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
         </profileDesc>
         <revisionDesc>
             <change who="DR" when="2018-04-16">Created entity</change>
+            <change who="DR" when="2020-04-29">Added information, included data from deleted PRS12136GabraM, PRS12702MikailHabashi</change>
         </revisionDesc>
     </teiHeader>
     <text xml:base="https://betamasaheft.eu/">
         <body>
             <listPerson>
                 <person sex="1">
-                    <persName xml:lang="gez" xml:id="n1">ገብረሚካኤል፡</persName>
+                    <persName xml:lang="gez" xml:id="n1">ገብረ፡ ሚካኤል፡</persName>
                     <persName xml:lang="gez" type="normalized" corresp="#n1">Gabra Mikāʾel</persName>
+                    <persName xml:lang="ar" xml:id="n2">ميخائيل الحبشي</persName>
+                    <persName xml:lang="ar" type="normalized" corresp="#n2">Mīhāʾīl al-Ḥabašī</persName>
+                    <persName><roleName type="title">qummuṣ</roleName></persName>
+                    <occupation type="ecclesiastic">A monk and consecrated priest who resided in
+                        <placeName ref="INS0478DS">Dayr as-Suryān</placeName> in the
+                        19th or early 20th century.</occupation>
+                    <residence>He is associated with <placeName ref="INS0354DB"/> and 
+                        <placeName ref="INS0352EA"/> in the manuscripts attesting him.</residence>
+                    <floruit notBefore="1875" notAfter="1960"/>
+                    <note>
+                        <p>The modern Arabic manuscript description placed on a paper sheet inside <ref type="mss" corresp="DSEthiop13"/> states
+                        that Mīḫāʾīl al-Ḥabašī was born in <placeName ref="LOC1178AddiNa"/> in Eritrea and that his mother's name was
+                        <foreign xml:lang="ar">Lītā Māryām</foreign>, possibly a transcription of <foreign xml:lang="gez">Walatta Māryām</foreign>.</p>
+                        
+                      <p>  The <ref target="https://stmaryrodelfarag.com/%D8%A7%D8%A8%D8%A7%D8%A1-%D8%B3%D8%A7%D8%A8%D9%82%D9%8A%D9%86/">website of the 
+                            church St Mary in Rawḍ al-Farag in Cairo</ref> states that a <foreign xml:lang="ar">qummuṣ</foreign> Mīḫāʾīl al-Ḥabašī lived in this church, until he was ordered by
+                        <persName ref="PRS3260CyrilVI"/> to return to a monastery. Since it was impossible for him to return to Ethiopia, he went to <placeName ref="INS0478DS"/>, 
```

Comment:
to a monastery - Dayr as-Suryan or his Ethiopian monastery? (then better to "his monastery in his native country"). "To return to Eritrea", then, or just "to return"

Reply:
To erase his mother's name - I think it is possible, for instance if he inherited the book or she just gave it to as present him.

Reply:
Ah yes, I misunderstood the Arabic. It was apparently a general order (I'm trying to find a date now), not specific to this monk.